oci/oci.go

```diff
@@ -561,7 +561,7 @@ func (r *Runtime) UpdateContainer(c *Container, res *rspec.LinuxResources) error
 	return nil
 }
 
-func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration) error {
+func waitContainerStop(ctx context.Context, c *Container, timeout time.Duration, forceKill bool) error {
```

Comment:
PR looks ok, but afaict it's not actually force killing anything right? maybe this should be ignoreKill (just nitpicking)

Reply:
Sure, I can change the name here.

Reply:
Done !